.github/workflows/release.yml

```diff
@@ -27,13 +27,8 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 20
-      - id: install-zx
-        run: npm i -g zx
-      - id: install-semver-tool
-        run: |
-          wget -O /usr/local/bin/semver https://raw.githubusercontent.com/fsaintjacques/semver-tool/master/src/semver
-          chmod +x /usr/local/bin/semver
-      - run: zx ./release.mjs -v $VERSION_TO_BUMP
+      - name: Install dependencies and run release script
+        run: yarn add -D -E zx@8.1.4 semver@7.6.3 && yarn zx ./release.mjs -v $VERSION_TO_BUMP
```

Comment:
Might be more readable to split this into 2 lines you can use a | to separate them

Reply:
Nice, done 👍